index.html

```diff
@@ -8,6 +8,7 @@
   </head>
   <body>
     <div id="root"></div>
-    <script type="module" src="/src/main.tsx"></script>
+    <script type="module" src="/src/main.tsx">
+    </script>
```

Comment:
Is it necessary to change the identation here? Could you rollback to the previous state?

Reply:
You're right, I'll undo it in a moment.